tracing-forest/src/lib.rs

```diff
@@ -39,12 +39,37 @@
 //! ```
 //! Then, add [`tracing_forest::init`](crate::init) to your main function:
 //! ```
-//! # #[allow(clippy::needless_doctest_main)]
 //! fn main() {
+//!     // Initialize a default `ForestLayer` subscriber
 //!     tracing_forest::init();
 //!     // ...
 //! }
 //! ```
+//! This crate also provides tools for much more advanced configurations:
+//! ```
+//! use tracing_forest::{Processor, ForestLayer};
+//! use tracing_subscriber::{Registry, layer::SubscriberExt};
```

Comment:
*banging on table* prelude, prelude, prelude ... :P

Reply:
Yes yes I know lol... I will get around to your PR soon (oops).